src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -237,6 +235,10 @@ export const TimelineNav = ({
     } else if ([L_ARROW_KEYCODE, U_ARROW_KEYCODE].includes(e.key)) {
       /* 4 */
       timelineNavItemRefs[prev].current.focus();
+    } else if (e.key === ENTER_KEYCODE) {
+      setTimeout(() => {
+        backRef.current?.focus(); /* 5 */
+      }, 500);
```

Comment:
Oh nice solution! I think we also need spacebar though?

Reply:
Thanks! Added.